### ISSUE TITLE
Beatsaber version 1.15.0 not listed

### ISSUE THREAD

**Issue by mhombach** [Assistant/ModAssistant#332]
ModAssistant is not listing me correctly my installed version 1.15.0:

![image](https://user-images.githubusercontent.com/34406120/117063489-32a4fa00-ad25-11eb-9cec-132914769ffa.png)

Not sure if this is a bug on my side, but I did the regular Beatsaber-update (Windows 10, 64 bit, SteamVR/Valve Index) so I have to assume it's not on my side.

**Comment by Melopod** [comment]
That would be because 1.13.4 mods work on versions 1.13.4, 1.13.5, 1.14.0, and 1.15.0

**Comment by mhombach** [comment]
Hm... ok, got it. Although this can get very irritating. How should I know the difference between my newly installed version being compatible with the mods or ModAssistant not beeing updatet yet? :/
I would favour having the installed version listed, so that I know that ModAssistant has recognized my installed version correctly and so does also correctly display the updated status of each mod. But if this is too much to ask for, I guess we have to deal with it as it is right now.

**Comment by Melopod** [comment]
You would know simply because Mod Assistant would have a new version listed for the most recent version. Versions don't change on MA because mods haven't changed and would otherwise need to be re-uploaded just to change game version numbers, when there's no need.
If mods of a previous version are no longer compatible with a recent version, mods will then have to be updated and uploaded for the most recent version and MA will then have an option for the most recent version of the game.
tldr: if there isn't an option for the most recent version of the game, it means mods for previous versions still work.

**Comment by mhombach** [comment]
I got you, I'm not attacking that idea/pattern :)
I was just concerned about if what you wrote is 100% clear to the normal/new user of ModAssistant.
For example: It could be more easy to understand if it would look like this:

![image](https://user-images.githubusercontent.com/34406120/117114696-77b04700-ad8c-11eb-99c6-42899ec2a6fa.png)

**Comment by Graftax** [comment]
I had this exact some confusion, and the proposed change would have helped me understand what was happening, immediately.

**Comment by chriswen** [comment]
Interesting, as a new beatsaber user, my original assumption was that the mods weren't compatible with Version 1.15

**Comment by vadixidav** [comment]
@chriswen I have used beatsaber mods on and off for a while, and it confused me too! Good thing I came to the issue tracker. I think a version range might be more helpful. Thanks above.
